Modules/TRD/src/DigitsTask.cxx

```diff
@@ -146,16 +146,12 @@ void DigitsTask::monitorData(o2::framework::ProcessingContext& ctx)
     mNoiseMap = ptr.get();
   }
 
-  if (!mChamberStatus) {
-    auto ptr = ctx.inputs().get<std::array<int, MAXCHAMBER>*>("fedChamberStatus");
-    mChamberStatus = ptr.get();
-    // LB: only draw in plots if it is first instance, e.g. null ptr to non null ptr
-    if (mChamberStatus) {
-      // LB: no half chamber distribution map for Digits, pass it as null pointer
-      TRDHelpers::drawChamberStatusOnHistograms(mChamberStatus, nullptr, mLayers, NCOLUMN);
-    } else {
-      ILOG(Info, Support) << "Failed to retrieve ChamberStatus, so it will not show on plots" << ENDM;
-    }
+  // LB: Chamber Status cannot be loaded only once, it should always check object
+  auto ptr = ctx.inputs().get<std::array<int, MAXCHAMBER>*>("fedChamberStatus");
+  mChamberStatus = ptr.get();
+  if (mChamberStatus) {
+    // LB: no half chamber distribution map for Digits, pass it as null pointer
+    TRDHelpers::drawChamberStatusOnHistograms(mChamberStatus, nullptr, mLayers, NCOLUMN);
```

Comment:
Hi @Barthelemy, thanks. I am not sure, but maybe it would be better to keep previous setup and then reset `mChamberStatus` elsewhere/after a couple of instances.

I'll wait for @martenole input before committing changes

Reply:
This is indeed too heavy. You would draw on the histograms for every input data, independent of whether or not the CCDB object was updated. You should instead create a callback to `finaliseCCDB`. See slide 5 of https://docs.google.com/presentation/d/1WPNixo57zAn_gM6vMCIBOHs-DZqcKAkLX0CwUgO_VYg/edit?pli=1#slide=id.g112eef84963_0_3.
It should look something like
```c++
void DigitsTask::finaliseCCDB(o2::framework::ConcreteDataMatcher& matcher, void* obj)
{
  if (matcher == ConcreteDataMatcher("TRD", "FCHSTATUS", 0)) {
    TRDHelpers::drawChamberStatusOnHistograms(static_cast<std::array<int, MAXCHAMBER>*>(obj), nullptr, mLayers, NCOLUMN);
  }
}
```

Reply:
Nice! Implemented.